webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import {GitPullRequestIcon, IssueOpenedIcon, IconProps} from '@primer/octicons-react';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+import {Badge, Tooltip, OverlayTrigger} from 'react-bootstrap';
+import * as CSS from 'csstype';
+
+import CrossIcon from 'src/images/icons/cross';
+import DotIcon from 'src/images/icons/dot';
+import TickIcon from 'src/images/icons/tick';
+import SignIcon from 'src/images/icons/sign';
+import {formatTimeSince} from 'src/utils/date_utils';
+import {GitlabItemsProps, Label} from 'src/types/gitlab_items';
+
+export const notificationReasons: Record<string, string> = {
+    assigned: 'You were assigned to the issue/merge request',
+    review_requested: 'You were requested to review a merge request.',
+    mentioned: 'You were specifically @mentioned in the content.',
+    build_failed: 'GitLab build was failed.',
+    marked: 'Task is marked as done.',
+    approval_required: 'Your approval is required on this issue/merge request.',
+    unmergeable: 'This merge request can not be merged.',
+    directly_addressed: 'You were directly addressed.',
+    merge_train_removed: 'A merge train was removed.',
```

Comment:
This string's meaning is unclear. Under what circumstances would a user see this message? Please define what a merge train is.

Reply:
Merge train is a GitLab feature in which we can add our merge requests in queue, which runs a pipeline to verify that the changes can merge into the default branch.
You can read more about merge train here: https://docs.gitlab.com/ee/ci/pipelines/merge_trains.html

Reply:
Thank you!